### ISSUE TITLE
Upgrade to React 17

### ISSUE THREAD

**Issue by michael-arvanitidis** [ugogo/react-input-verification-code#39]
I'm using React 17 and getting this error when trying to install reaact-input-verification-code

```
npm ERR! While resolving: admin@0.1.0
npm ERR! Found: react@17.0.2
npm ERR! node_modules/react
npm ERR!   react@"^17.0.1" from the root project
npm ERR!
npm ERR! Could not resolve dependency:
npm ERR! peer react@"^16.0.0" from react-input-verification-code@0.8.0
npm ERR! node_modules/react-input-verification-code
npm ERR!   react-input-verification-code@"*" from the root project
```

Is there a reason for not making this package compatible with react 17?

**Comment by ugogo** [comment]
Hey @michael-arvanitidis, thanks for raising the issue
I'll have a look 🙏

**Comment by ugogo** [comment]
👋 @michael-arvanitidis sorry for the delay
This is fixed in the [`0.9.0`](https://github.com/ugogo/react-input-verification-code/releases/tag/0.9.0) version

Let me know if you have any other issues

**Comment by michael-arvanitidis** [comment]
@ugogo no worries, thanks for the help!
